### ISSUE TITLE
Add per-park min_training_year config to prevent stale data poisoning

### ISSUE THREAD

**Issue by hazeydata** [hazeydata/theme-park-crowd-report#48]
## Problem

AK training data is 74.8% pre-2016 (213K rows) with avg wait of 12.2 min. Post-2016 AK averages 27.2 min — a completely different park (full-day ops vs half-day). The 2014 data alone is 183K rows (64% of all AK data) at 12.3 min avg. This stale data is dragging AK predictions down by ~11 WTI points.

MK has a similar pre/post ratio (83.5% pre-2016) but the avg wait gap is much smaller (16.3 vs 19.4), so the impact is less severe.

## Proposal

1. Add `min_training_year` to pipeline config with per-park overrides
2. Default: no cutoff (backward compatible)
3. AK override: 2016 (removes the half-day era)
4. Review other parks: IA and UH may also benefit from cutoffs
5. Training step (s07) reads this config and filters training data accordingly

## Data

```
AK Training Data Split:
  Pre-2016: 213,741 rows (74.8%) — avg wait 12.2 min
  Post-2016: 72,016 rows (25.2%) — avg wait 27.2 min

  2014 alone: 183,189 rows — 64% of all AK data
```

## Future: Recency Weighting (Phase 2)

Hard cutoff is the quick fix. Longer term, per-park recency weighting (exponential decay on older data) is more elegant and handles gradual operational changes.

## Owner
Bam-Bam

## Priority
🔴 Phase 1

**Comment by hazeydata** [comment]
## Investigation Update — Root Cause Confirmed

**Fred identified the source:** In 2014, Disney began publishing actual wait times through their official API alongside posted times. The TouringPlans scraper collected both at ~7-minute intervals. Disney eventually discontinued the actual-time API, but the data remained in the database.

### Evidence

```
2014 AK07, Jan 12 — matched pairs at same timestamp:
08:56:04  ACTUAL  9 min    POSTED  0 min
09:00:06  ACTUAL  9 min    POSTED  0 min
09:14:03  ACTUAL  4 min    POSTED  0 min
```

### Data Quality Issues
- 2014 ACTUAL avg: 12.3 min vs POSTED avg: 19.4 min — the API data systematically reported lower waits
- 45.6 obs/entity/day in 2014 vs 1.7 in modern years (25x oversampling)
- POSTED shows 0 while ACTUAL shows 9 — clearly different data streams with questionable accuracy
- 2014-2015 = **72.1% of all AK ACTUAL training data**

### Revised Fix
A `min_training_year: 2016` cutoff is the clean fix. It removes:
1. High-frequency Disney actual-time API contamination (2014-2015)
2. Half-day AK operating era
3. Questionable data quality Fred remembered from that era

All WDW parks affected (EP: 45.4, MK: 35.5, HS: 24.4 obs/entity/day in 2014). The cutoff should apply to all WDW parks, not just AK.
